templates/cook/cook_list.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  <div class="container my-5">
+    <div class="d-flex justify-content-between align-items-center mb-4">
+      <h1>Cooks</h1>
+      <a href="{% url 'management_system:cook-create' %}" class="btn btn-primary">Create Cook</a>
+    </div>
+
+    {% if cook_list %}
```

Comment:
if cook_list is not provided just display an empty table

Reply:
Removed if else statement.